ui/src/shared/apis/query.test.ts

```diff
@@ -0,0 +1,381 @@
+import {mocked} from 'ts-jest/utils'
+import {runQuery} from 'src/shared/apis/query'
+
+import {RunQuerySuccessResult} from 'src/shared/apis/query'
+import {AppState} from 'src/types'
+
+jest.mock('src/shared/apis/query')
+
+const mockState = ({
+  app: {
```

Comment:
do we need this?

Reply:
We use this at some point in the process of creating of a query. The tests were complaining when this wasn't included